gpiozero/pins/mock.py

```diff
@@ -413,3 +385,34 @@ def tx_word(self, value, bits_per_word=None):
             bits = reversed(bits)
         self.tx_buf.extend(bits)
 
+
+class MockFactory(LocalPiFactory):
+    def __init__(self, revision='a21041', pin_class=MockPin):
+        super(MockFactory, self).__init__()
+        self._revision = revision
+        self.pin_class = pin_class
+
+    def _get_address(self):
+        return ('mock',)
+
+    def _get_revision(self):
+        return self._revision
+
+    def reset(self):
```

Comment:
I wonder if this should be named `_reset`, given that it's a method that no other factories have?